themes-default/slim/static/js/home/display-show.js

```diff
@@ -414,4 +424,19 @@ MEDUSA.home.displayShow = function() { // eslint-disable-line max-lines
             log.error(error.data);
         });
     });
+
+    function setQuality(quality, seriesSlug, episodes) {
+        const patchData = {};
+        episodes.forEach(episode => {
+            patchData[episode] = { quality: parseInt(quality, 10) };
+        });
+
+        api.patch('series/' + seriesSlug + '/episodes', patchData)
+            .then(response => {
```

Comment:
`.then`  should be on the same line as the `api.patch`.

Reply:
If I change it the XO lint won't pass.

Reply:
Try
```js
api.patch('series/' + seriesSlug + '/episodes', patchData).then(response => {
    log.info(response.data);
    window.location.reload();
}).catch(error => {
    log.error(error.data);
});
```